content/docs/community/release_checklist.md

```diff
@@ -44,6 +44,20 @@ export RELEASE_BRANCH_NAME="release-X.Y"
 export RELEASE_CANDIDATE_NAME="$RELEASE_NAME-rc1"
 ```
 
+We are also going to be adding security and verification of the release process by
+hashing the binaries and providing signature files. We perform this using
+[GitHub and GPG](https://help.github.com/en/articles/about-commit-signature-verification).
+If you do not have GPG already setup you can follow these steps:
```

Comment:
There needs to be an empty line prior to the list or else it won't render properly

Reply:
@lucperkins thanks for catching that. I updated the formatting there. Does it look ok now?

Reply:
Yep! https://deploy-preview-252--helm-v3-dev.netlify.com/docs/community/release_checklist/